ldclient.go

```diff
@@ -583,13 +554,18 @@ func (client *LDClient) SecureModeHash(context ldcontext.Context) string {
 // this does not guarantee that the flags are up to date; if you need to know its status in more detail, use
 // [LDClient.GetDataSourceStatusProvider].
 //
```

Comment:
Added this comment because that's what our unit tests expect. Didn't see it documented.